apis/v1alpha2/httproute_types.go

```diff
@@ -487,6 +484,10 @@ type HTTPRouteFilter struct {
 	// Implementers are encouraged to define custom implementation types to
 	// extend the core API with implementation-specific behavior.
 	//
+	// If a reference to a custom filter type cannot be resolved, the filter
+	// MUST NOT be skipped. Instead, requests that would have been processed by
+	// that filter SHOULD receive a HTTP error response.
```

Comment:
Why would they not do this? If it's only because the implementation doesn't support it, then we should say "MUST, except if".

Reply:
Yeah I think you're right, can't think of a reason not to use MUST here.